src/main/java/org/arl/fjage/connectors/WebSocketHubConnector.java

```diff
@@ -231,8 +231,11 @@ public void onClose(int statusCode, String reason) {
     }
 
     @OnWebSocketError
-    public void onError(Throwable t) {
-      log.warning(t.toString());
+    public void onWebSocketError(Throwable cause)  {
+      if (cause instanceof org.eclipse.jetty.io.EofException) {
+        log.info(cause.toString());
+      }
+      log.log(Level.WARNING, "WebSocket error: ", cause);
```

Comment:
Won't this log the error as WARNING as well as INFO? I think the original issue was that most of these `EofException` happen when the connection to a client (browser) is terminated while we're the server is half way sending something. The WARNING + Stacktrace just make them very verbose when all we need to log is "we lost connection to the client".